test/unit_tests/model/ImageTest.cpp

```diff
@@ -23,10 +23,15 @@ TEST(Image, testGetImageApplyOrientation) {
     // Image width is 500px and height 130px - but has exif data saying image should be
     // rotated 90 deg CW to have correct orientation.
     std::ifstream imageFile{GET_TESTFILE("images/r90.jpg"), std::ios::binary};
-    auto imageData = std::string(std::istreambuf_iterator<char>(imageFile), {});
+    imageFile.seekg(0, std::ios::end);
+    const std::streamsize size = imageFile.tellg();
```

Comment:
I think, tellg is not valid to determine the size of the stream. We should use the filesystem for that.

Reply:
Fixed. Now use `std::filesystem::file_size`